GeosCore/fullchem_mod.F90

```diff
@@ -1574,6 +1588,9 @@ SUBROUTINE Do_FullChem( Input_Opt,  State_Chm, State_Diag,                 &
        RETURN
     ENDIF
 
+    print*, '### bmystop', failed2x
```

Comment:
Could you remove these lines? I think they must have been for testing?

Reply:
Now fixed in commit 12a1acf32f502df733775a033f391640f5e203b1